testing/integration_test.go

```diff
@@ -62,7 +62,7 @@ apt-get -y -q install git build-essential  >/dev/null
 curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash >/dev/null
 export NVM_DIR="$HOME/.nvm" >/dev/null
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" >/dev/null
-nvm install {{.NodeVersion}} >/dev/null
+nvm install {{.NodeVersion}} &>/dev/null
```

Comment:
It would be good to add a comment on why both stdout and stderr are redirected for this command.

Reply:
Done.